storage/remote/otlptranslator/prometheusremotewrite/histograms_test.go

```diff
@@ -30,7 +30,7 @@ import (
 
 	"github.com/prometheus/prometheus/prompb"
 
-	prometheustranslator "github.com/prometheus/prometheus/storage/remote/otlptranslator/prometheus"
+	prometheustranslator "github.com/prometheus/otlp-prometheus-translator"
```

Comment:
The linter should have caught it, but this import should be moved to the block starting at line 25, i.e. non-stdlib/non-local imports.